ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology-mpls-vpn.md

```diff
@@ -0,0 +1,261 @@
+        <node inventory hostname>:
+          # Vars defined per node
+  nodes:
+    <node inventory hostname>:
+      # Vars defined per node
+
+      # Unique identifier | Required.
+      id: < integer >
+
+      # Node management IP address | Optional.
+      mgmt_ip: < IPv4_address/Mask >
+```
+
+## Node Variables details
+
+### Generic configuration management
+
+```yaml
+< node_type_key >:
+
+  defaults:
+    # Arista platform family | Required.
+    platform: < Arista Platform Family >
+
+    # Rack that the switch is located in (only used in snmp_settings location) | Optional
+    rack: < rack_name >
+
+    # EOS CLI rendered directly on the root level of the final EOS configuration | Optional
+    raw_eos_cli: |
+      < multiline eos cli >
+
+    # Custom structured config for eos_cli_config_gen | Optional
+    structured_config: < dictionary >
+```
+
+### Uplink management
+
+Unlike with the l3ls-evpn design type, underlay p2p links are built using the backbone_interfaces dictionary:
+
+```yaml
+backbone_interfaces:
+  p2p_links_ip_pools:
+    < pool name >: < IPv4_address/Mask >
+  p2p_links_profiles:
+    < backbone profile name >:
+      speed: < speed >
+      mtu: < mtu >
+      isis_hello_padding: < true | false >
+      isis_metric: < metric >
+      ip_pool: < pool name >
+      isis_circuit_type: < isis circuit type >
+      ipv6_enable: < true | false >
+  p2p_links:
+    - id: < Link ID, used for selecting a subnet from the provided pool >
+      nodes: ['< node1 inventory_hostname >', '< node2 inventory_hostname >']
+      interfaces: ['< node1 interface >', '< node2 interface >']
+      profile: < backbone profile name >
+    - id: < Link ID >
+      ...
+```
+
+#### ISIS underlay protocol management
+
+```yaml
+< node_type_key >:
+
+  defaults:
+    # isis system-id prefix (4.4 hexadecimal)
+    isis_system_id_prefix: < hhhh.hhhh >
+
+    # Number of path to configure in ECMP for ISIS
+    isis_maximum_paths: < integer >
+
+    # Base value for ISIS-SR Node-SID
+    node_sid_base: 100
+
+    # Node is-type as configured under the router isis instance.
+    is_type: level-2
+```
+
+### Loopback and ISIS-SR Node-SID management
+
+```yaml
+< node_type_key >:
+
+  defaults:
+    # IPv4 subnet for Loopback0 allocation
+    loopback_ipv4_pool: < IPv4_address/Mask >
+
+    # Offset all assigned loopback IP addresses.
+    loopback_ipv4_offset: 2
+
+    # IPv6 subnet for Loopback0 allocation
+    loopback_ipv6_pool: < IPv6_address/Mask >
+
+    # Offset all assigned loopback IP addresses.
+    loopback_ipv6_offset: 2
+```
+
+### BGP & EVPN Control plane
+
+```yaml
+< node_type_key >:
+
+  defaults:
+
+    # List of EOS command to apply to BGP daemon | Optional
+    bgp_defaults: [ < List of EOS commands> ]
+
+    # Acting role in EVPN control plane.
+    # Override role definition from node_type_keys
+    # Can be set per node
+    overlay_role: < client | server | none | Default -> refer to node type variable table >
```

Comment:
so here you mention `overlay_role` which is more in line with what I mentioned in the other comment. Description is wrong (mentioning EVPN). Please align everywhere. It could be useful to have different roles for evpn and mpls (for gateways), so maybe just call this `mpls_overlay_role`

Reply:
mpls_overlay_role is implemented and doc updated.